script/build_all_plugins_app.sh

```diff
@@ -11,6 +11,7 @@ source "$SCRIPT_DIR/common.sh"
 check_changed_packages > /dev/null
 
 readonly EXCLUDED_PLUGINS_LIST=(
+  "flutter_plugin_android_lifecycle"
```

Comment:
Adding this to the exclude list because the all plugins app creates a local dependency while this plugin creates a hosted dependency. This caused errors at compilation.